client/src/components/AddToCartRegion.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+var AddToCartRegion = function(props) {
+  return (
+    <React.Fragment>
```

Comment:
100% a nit because it's more just for if you don't know: instead of React.Fragment, you can just use `<>` and `</>` for the same effect

Reply:
This one I'm inclined to leave in; I like the explicitness of <React.Fragment>. 
It's saying loud and clear: "I'm wrapping this in a single DOM element because react demands it, not because this should actually be a single DOM element. I babel to clean this back up so it never makes it to the client computer."